### ISSUE TITLE
Cant switch off (or on) interfaces/

### ISSUE THREAD

**Issue by Slona** [elad-bar/ha-edgeos#145]
Sorry i dont know which logs or statistics should i attach, but the issue is the following: when i try to disable or enable interface via HA UI then it toggles off and then in a few seconds goes back to its initial status. And there is nothing in logbook

https://github.com/elad-bar/ha-edgeos/assets/928132/ef89cd3c-3f32-42ff-a3ed-3ca3d134b339

A user is admin, i checked that
![image](https://github.com/elad-bar/ha-edgeos/assets/928132/5ffea78c-8f20-4c04-80a7-14c93378ae7a)

**Comment by elad-bar** [comment]
Can you pls add logs with debug level?

Thanks

**Comment by elad-bar** [comment]
In addition, which version og ha/integration are you using?

**Comment by Slona** [comment]
[home-assistant_2024-05-10T13-05-26.841Z.log](https://github.com/elad-bar/ha-edgeos/files/15275433/home-assistant_2024-05-10T13-05-26.841Z.log)

I have cut the log to last 5 minutes (tried ro disable interface 4 times)

The versios are last available:
Core
2024.5.2
Frontend
20240501.1

ha-edgeos
v2.1.3

**Comment by Slona** [comment]
I guess that all information available in logs regarding this issue is the following:

```
2024-05-10 16:05:06.909 DEBUG (MainThread) [custom_components.edgeos.managers.coordinator] Disable interface eth3
2024-05-10 16:05:06.909 INFO (MainThread) [custom_components.edgeos.managers.rest_api] Set state of interface eth3 to False
2024-05-10 16:05:06.910 WARNING (MainThread) [custom_components.edgeos.managers.rest_api] Request failed, Endpoint: set, Error: set, Line: 266
2024-05-10 16:05:06.910 ERROR (MainThread) [custom_components.edgeos.managers.rest_api] Failed to set state of interface eth3 to False
```

**Comment by elad-bar** [comment]
fixed in [v2.1.6](https://github.com/elad-bar/ha-edgeos/releases/tag/v2.1.6), pls check

thanks

**Comment by Slona** [comment]
> fixed in [v2.1.6](https://github.com/elad-bar/ha-edgeos/releases/tag/v2.1.6), pls check
> 
> thanks

Oh God, it works! Thanks so much!

You're breathtaking!
